lunes_cms/cms/models/static.py

```diff
@@ -24,6 +24,9 @@ class Static:
         ("Nomen", "Substantiv"),
         ("Verb", "Verb"),
         ("Adjektiv", "Adjektiv"),
+        ("Numeral", "Numeral"),
+        ("Pronomen", "Pronomen"),
+        ("Adverbien", "Adverbien"),
```

Comment:
```suggestion
        ("Adverb", "Adverb"),
```
Since we used singular for all other word types, probably we should use it for this as well?

Reply:
Yes!